mitxgraders/helpers/calc/expressions.py

```diff
@@ -1,7 +1,9 @@
 """
-calc.py
+expressions.py
 
-Defines classes to parse and evaluate mathematical expressions.
+Defines classes to parse and evaluate mathematical expressions. Implements
+similar functionality and API as edX's calc.py, but re-written with enhancements
+and to betterseparate parsing and evaluation.
```

Comment:
space between better and separate

Reply:
you didn't want to tell me to better separate those words?

Reply:
done :)